docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -347,24 +382,27 @@ Next, users should edit the new ``config.yaml`` file to customize it for their m
       EXPT_SUBDIR: test_community
    task_get_extrn_ics:
       USE_USER_STAGED_EXTRN_FILES: true
-      EXTRN_MDL_SOURCE_BASEDIR_ICS: "/path/to/UFS_SRW_App/develop/input_model_data/<model_type>/<data_type>/<YYYYMMDDHH>"
+      EXTRN_MDL_SOURCE_BASEDIR_ICS: /path/to/UFS_SRW_App/v2p2/input_model_data/<model_type>/<data_type>/${yyyymmddhh}
```

Comment:
This should be: `/path/to/UFS_SRW_data/v2p2/input_model_data/<model_type>/<data_type>/${yyyymmddhh}`
Awhile back we changed our staged data location from `UFS_SRW_App` to `UFS_SRW_data`.

This goes for the `EXTRN_MDL_SOURCE_BASEDIR_LBCS` variable as well.